dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -169,17 +183,31 @@ export const ContainerLayout = ({
 					stretchRight={stretchRight}
 					format={format}
 				>
-					<Hide when="above" breakpoint="leftCol">
-						<ContainerTitle
-							title={title}
-							fontColour={fontColour || overrides?.text.container}
-							description={description}
-							url={url}
-							showDateHeader={showDateHeader}
-							editionId={editionId}
-						/>
-					</Hide>
-					{children}
+					<div css={headlineContainerStyles}>
+						<Hide when="above" breakpoint="leftCol">
+							<ContainerTitle
+								title={title}
+								fontColour={
+									fontColour || overrides?.text.container
+								}
+								description={description}
+								url={url}
+								showDateHeader={showDateHeader}
+								editionId={editionId}
+							/>
+						</Hide>
+						{toggleable && sectionId !== undefined && (
```

Comment:
Nice to have but, can we introduce a type that requires a sectionId to be set when toggleable is true?

Reply:
Not sure typescript has a way to do this but I can have a poke about.

Reply:
I don't think theres a good / easy way to do this, feel free to raise a follow up PR though!

Reply:
I think the `Island` component has something like this in it. But to do this it needed multiple sets of props though, which added quite a lot of complexity.

Not sure if there's a better way but I bet @mxdvl knows 😉